### ISSUE TITLE
Products Detail on order in Cybersource

### ISSUE THREAD

**Issue by lemusi** [mmabingo/WC_CyberSourceSecureAcceptance_SOP#4]
Hi again Mikochi,

Another thing my Bank is asking me to do with cybersource is to list the products that are being purchased through the order.  

Do you have any suggestions on how to accomplish this?

They say they need this data to be setup:

<span>item_0_code:</span><input type="text" name="item_0_code" size="25"><br/>
<span>item_0_name:</span><input type="text" name="item_0_name" size="25"><br/>
<span>item_0_quantity:</span><input type="text" name="item_0_quantity" size="25"><br/>
<span>item_0_sku:</span><input type="text" name="item_0_sku" size="25"><br/>
<span>item_0_unit_price:</span><input type="text" name="item_0_unit_price" size="25"><br/>

//Si hay más de un elemento en el shopping cart es necesario iterar el ítem_0 por los consecutivos:
       Ítem_1
       Ítem_2 etc.

Let me know what you think,

Thanks.

Ivan

**Comment by JeansReal** [comment]
Have you solve this?

**Comment by lemusi** [comment]
Yes I have.  I can give it to you for a fair price..  please let me know if you are interested.
